Resources/Prototypes/DeltaV/Entities/Mobs/NPCs/glimmer_creatures.yml

```diff
@@ -0,0 +1,84 @@
+- type: entity
+  name: glimmer mite
+  parent: SimpleMobBase
```

Comment:
is there a reason this cant parent mouse

Reply:
Agree with Deltanedas here, seems like that'd save a lot of trouble. Unless theres a reason why it cant